packages/stark-build/config/webpack.common.js

```diff
@@ -466,21 +465,21 @@ module.exports = function (options) {
 				},
 				platform: 0, // 0 = browser, 1 = server
 				compilerOptions: {
+					// FIXME Those options force AngularCompilerOptions. Cannot be overwritten by angularCompilerOptions defined in tsconfig.json
+					generateCodeForLibraries: true,
```

Comment:
As we discussed, maybe we could simply read these options from the tsconfig.json file of the build (or the file that we will always take as basis for the rest) so we can avoid some duplication and prevent some errors in case we forget to change this options later on

Reply:
Makes sense, we should do that

Reply:
@SuperITMan can you adapt the PR for this?